### ISSUE TITLE
Crash on latest Audio version

### ISSUE THREAD

**Issue by codlab** [voxeet/voxeet-sdk-android-audio#1]
When having bluetooth disconnecting or reconnecting multiple times (possible system lifecycle), the bluetooth implementation of the library could crash in the audio v2.

```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void android.os.Handler.removeCallbacks(java.lang.Runnable)' on a null object reference
       at com.voxeet.audio2.manager.BluetoothHeadsetDeviceManager.onNewBluetoothServiceConnectivity(BluetoothHeadsetDeviceManager.java:127)
       at com.voxeet.audio2.manager.BluetoothHeadsetDeviceManager.lambda$6wqK44RXdYIe7s4vy9u-gr34RVo(BluetoothHeadsetDeviceManager.java)
       at com.voxeet.audio2.manager.-$$Lambda$BluetoothHeadsetDeviceManager$cGuJelgeSC4-1iRlROCOl6EjarU.apply(-.java:4)
       at com.voxeet.audio2.manager.bluetooth.BluetoothHeadsetServiceListener$1.onServiceDisconnected(BluetoothHeadsetServiceListener.java:48)
       at android.bluetooth.BluetoothHeadset$3.handleMessage(BluetoothHeadset.java:1206)
       at android.os.Handler.dispatchMessage(Handler.java:107)
       at android.os.Looper.loop(Looper.java:214)
       at android.app.ActivityThread.main(ActivityThread.java:7356)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:492)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:930)
```

The expected behaviour is to simply avoid the mechanism to be resending the command

**Comment by codlab** [comment]
Tracked as #106

**Comment by codlab** [comment]
available in v2.0.2

`compile "com.voxeet.sdk:audio:2.0.2"`
or
`implementation "com.voxeet.sdk:audio:2.0.2"`
